pkgs/development/python-modules/naked/default.nix

```diff
@@ -0,0 +1,57 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, unittestCheckHook
+, requests
+, nodejs
+, ruby
+, nix-update-script
+}:
+
+buildPythonPackage rec {
+  pname = "naked";
+
+  version = "0.1.32";
+
+  src = fetchFromGitHub {
+    owner = "chrissimpkins";
+    repo = "naked";
+    rev = "v${version}";
+    hash = "sha256-KhygnURFggvUTR9wwWtORtfQES8ANd5sIaCONvIhfRM=";
+  };
+
+  postPatch = ''
+  rm tests/test_NETWORK* -rf # no internet in sandbox
+
+  # TODO: disable only specific broken tests
+  rm tests/test_SYSTEM.py
+  rm tests/test_TYPES.py
+  rm tests/test_SHELL.py
+
+  # fix hardcoded absolute paths
+  substituteInPlace **/*.* \
+    --replace /Users/ces/Desktop/code/naked /build/source
+  '';
+
+  # doCheck = false;
+
+  preCheck =''
+    cd tests
+
+    PATH=$PATH:$out/bin
+  '';
+
+  propagatedBuildInputs = [ requests ];
+
+  nativeCheckInputs = [ unittestCheckHook nodejs ruby ];
+
+  passthru.updateScript = nix-update-script {};
+
```

Comment:
```suggestion
```

Reply:
auto update doesn't work without this, I tested using that `maintainers/update.nix` script

Reply:
Just take a look at some of the r-ryantm PR out there, and you'll soon realize we don't need it.

Reply:
removed those lines from the packages